### PR TITLE
Partial Factorial rework/cruciform stat tweaks

### PR DESCRIPTION
**Pull request by faaaay**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Alters the Factorial litanies and tweaks cruciform stats a bit. 

**Litany Changes:**
- Removed Prayer of the Forge, the disk-summoning litany. It's not very useful, nor is it engaging to use.
- Replaced the Overclocked Sanctifier and Giga-Excruciator with Holy Oils and Crusader's Seal. Oils are equivalent to a Sanctifier, but work for non-powered tools; a Seal boosts tool/weapon damage at the cost of either fuel or power, if applicable. Solid mods, but you can still get only one per 90 minutes. Summoning mods no longer requires an altar; you're making them, not begging for them.
- Added Touch of the Engine-Speaker, which summons an omnitool similar to the psionic one, but with base stats of 30, no medical tool qualities, and no scaling.

**Stat Changes:**
- Lemiscate regen lowered from **1.5 to 1**. Lemniscates have four really, really solid litanies, and they don't need such a massive regen rate when the second-highest is 0.6. 
- Factorial regen raised from **0.2 to 0.4**, on par with Divisors.
- Monomial regen raised from **0.1 to 0.2**. Their litanies are fine, but having 30 more Faith than the baseline isn't enough to justify a 0.1 regen rate. It's still slow, but hopefully a fairer tradeoff.

Factorials have needed some tweaking for a while, because despite being the engineers of the church, their litanies don't tie into that. This allows them to _make_ things, rather than just ask for them at the altar.
	
<hr>
</details>

## Changelog
:cl:
add: Added Touch of the Engine-Speaker, a litany allowing Factorials to summon a temporary omnitool
add: Added he Crusader's Seal and Holy Oils, Factorial-exclusive litanies
del: Removed the Overclocked Sanctifier and Giga-Excruciator
del: Removed the disk-summoning Prayer of the Forge
balance: buffed factorial/monomial faith regen from 0.2>0.4 and 0.1>0.2, respectively
balance: nerfed lemniscate faith regen from 1.5>1
/:cl:

### CODE REVIEW

File: code/game/objects/items/weapons/tools/mods/mod_types.dm [hunk 1]
```diff
@@ -765,6 +766,55 @@
...
+	GUN_UPGRADE_RECOIL = 0.8,
+	GUN_UPGRADE_FIRE_DELAY_MULT = 1.2,
+	GUN_UPGRADE_MOVE_DELAY_MULT = 1.2,
+	GUN_UPGRADE_CHARGECOST = 0.8
+	)
+	I.gun_loc_tag = GUN_MECHANISM
+	I.prefix = "blessed"
+
+/obj/item/tool_upgrade/augment/crusader_seal
+	name = "crusader's seal"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/mods/mod_types.dm:799]:
```suggestion
	name = "zealous seal"
```

File: code/game/objects/items/weapons/tools/mods/mod_types.dm [hunk 2]
```diff
@@ -765,6 +766,55 @@
...
+	I.tool_upgrades = list(
+		UPGRADE_FORCE_MULT = 1.1,
+		UPGRADE_FUELCOST_MULT = 1.25
+	)
+	I.weapon_upgrades = list(
+		GUN_UPGRADE_PVE_PROJ_MULT_DAMAGE = 1.1,
+		GUN_UPGRADE_CHARGECOST = 1.25,
+		UPGRADE_BULK = 0.5,
+		)
+	I.prefix = "crusader's"
```
Comment (Trilbyspaceclone) [inline code/game/objects/items/weapons/tools/mods/mod_types.dm:817]:
```suggestion
	I.prefix = "zealous"
```

File: code/modules/core_implant/cruciform/rituals/path.dm [hunk 3]
```diff
@@ -753,184 +753,133 @@
...
+		if(user.species?.reagent_tag != IS_SYNTHETIC)
+			if(user.nutrition >= nutri_cost)
+				user.nutrition -= nutri_cost
+			else
+				to_chat(user, SPAN_WARNING("You manage to cast the litany at a cost. The physical body consumes itself..."))
+				user.vessel.remove_reagent("blood",blood_cost)
+		set_personal_cooldown(user)
+		anti_cheat = FALSE
+		return TRUE
+	if (response == "Crusader's Seal")
```
Comment (Trilbyspaceclone) [inline code/modules/core_implant/cruciform/rituals/path.dm:803]:
```suggestion
	if (response == "Zealous Seal")
```

File: code/modules/core_implant/cruciform/rituals/path.dm [hunk 4]
```diff
@@ -753,184 +753,133 @@
...
-					user.vessel.remove_reagent("blood",blood_cost)
-			set_personal_cooldown(user)
-			anti_cheat = FALSE
-			return TRUE
-		if (response == "Cancel Litany")
-			fail("You decide not to obtain church artifice at this time.", user, C)
-			anti_cheat = FALSE
-			return FALSE
-	set_personal_cooldown(user)
+	var/response = input(user, "Which upgrade do you require?") in list("Holy Oils", "Crusader's Seal", "Nature's Blessing", "Cleansing Presence", "Faith's Shield", "Martyr's Gift", "Wrath of God", "Speed of the Chosen", "Cancel Litany")
```
Comment (Trilbyspaceclone) [inline code/modules/core_implant/cruciform/rituals/path.dm:791]:
```suggestion
	var/response = input(user, "Which upgrade do you require?") in list("Holy Oils", "Zealous Seal", "Nature's Blessing", "Cleansing Presence", "Faith's Shield", "Martyr's Gift", "Wrath of God", "Speed of the Chosen", "Cancel Litany")
```

File: code/game/objects/items/weapons/tools/mods/mod_types.dm (hunk 1 above)
Comment (faaaay) [reply code/game/objects/items/weapons/tools/mods/mod_types.dm:799]: Perhaps "righteous"?

Comment (Trilbyspaceclone) [reply code/game/objects/items/weapons/tools/mods/mod_types.dm:799]: Sure

**Comment by faaaay** [comment]
Okay, Crusader's has been replaced with Righteous.

**Pull request merged**